components/forms/BookDemo.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import Link from 'next/link';
+import JeanCircle from '../../public/img/people/jean-circle.png';
+import LeviCircle from '../../public/img/people/levi-circle.png';
+import ScottImage from '../../public/img/people/scottImage.jpg';
+
+
+const DemoForm = () => {
+  return (
+    <>
+      <div className="py-10">
+        <div className="flex justify-center pb-8">
+          <h1 className="inline-block m-0 md:text-4xl font-tuner lg:text-3xl md:text-2xl lg:leading-tight bg-gradient-to-br from-orange-400 via-orange-500 to-orange-600 bg-clip-text text-transparent">
+            Choose your location
+          </h1>
+        </div>
+        <div className="grid lg:grid-cols-2 gap-3 px-6 md:px-0 lg:px-6">
+          <div className="flex justify-center w-full items-center h-full">
+            <div className="w-full max-w-sm h-full">
+              <Link
+                href="https://meetings-eu1.hubspot.com/scott15?uuid=2f12f1a3-8b6a-4472-bfb5-2f3f930a1b19"
+                className="flex flex-col md:flex-row lg:flex-row w-full h-full items-center justify-between rounded-lg border border-input bg-background p-4 shadow transition transform duration-200 hover:scale-105 hover:bg-muted focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring text-blue-800 hover:text-blue-700"
+                prefetch={false}
+              >
+                <img
```

Comment:
use `next/image` `<Image />` component instead of normal `<img />`

Reply:
✅